examples/next/.gitignore

```diff
@@ -30,5 +30,7 @@ yarn-error.log*
 .env.test.local
 .env.production.local
 
-# vercel
-.vercel
+
+# amplify
+**/.amplify
+**/amplifyconfiguration*
```

Comment:
Think lines 33, 35 and 36 make sense but do we need the others?

Reply:
Hm im actually not sure why this changed, will just change it to add the amplify ignores